DataDog/storedog
# Correlate py traces and logs

## Description
This updates the ads and discount service to correlate traces + logs

https://datadoghq.atlassian.net/browse/WEB-3412

## How to test
1. Pull down branch locally
2. Build fresh versions of `ads` and `discounts` by running `docker-compose build ads discounts --no-cache`
3. Double check you have a local `.env` file at the root (follow instructions in README if you dont)
4. Start app via `docker-compose up`
5. Hit http://localhost a few times in your browser to generate logs
6. Open up DD, [view logs for ads and discounts](https://dd-corpsite.datadoghq.com/logs?query=service%3A%28discounts%20OR%20ads%29&cols=service&index=%2A&messageDisplay=inline&stream_sort=desc&viz=stream), click a log, verify that traces are populating in the context window

It should look like this
<img width="1391" alt="Screen Shot 2023-04-04 at 11 30 41 AM" src="https://user-images.githubusercontent.com/9119168/229842912-8eef339b-fef7-43b6-91b3-d849f0a658a6.png">
